rpm/systemd/diamond.service

```diff
@@ -2,8 +2,9 @@
 Description=diamond - A system statistics collector for graphite
 
 [Service]
-ExecStart=/usr/bin/python /usr/bin/diamond --log-stdout --foreground
-Restart=on-abort
+ExecStart=/usr/bin/python /usr/bin/diamond -p /var/run/diamond.pid
```

Comment:
@shortdudey123 didn't notice that, thanks.
we can remove "-p /var/run/diamond.pid" and leave PIDFile=/var/run/diamond.pid

although if someone changes the default pidfile and it doesn't change the systemd unit, service won't start.
maybe we should put a comment on the conf file and systemd unit? as far as I know I can't remove PIDFile option because systemd needs it to keep track of the service, if you remove it tries to guess but fails half the time, also, another type of unit like "simple" doesn't work because diamond forks when it starts (correct me if I am wrong).

let me know what you think.